### ISSUE TITLE
Enhanced setup check for libxml 2.7.0

### ISSUE THREAD

**Issue by kasi45** [nextcloud/server#205]
Nextcloud version: 9.0.50 (stable)

the loaded libxml is version 2.9.2 but nextcloud can not start.

example:
```
[~] # /mnt/ext/opt/apache/bin/php -i|grep libXML
libXML support => active
libXML Compiled Version => 2.6.30
libXML Loaded Version => 20902
libXML streams => enabled
```

https://github.com/nextcloud/server/blob/stable9/lib/private/util.php

```
if(function_exists('xml_parser_create') &&
	version_compare('2.7.0', LIBXML_DOTTED_VERSION) === 1) {
	$errors[] = array(
		'error' => $l->t('libxml2 2.7.0 is at least required. Currently %s is installed.', [LIBXML_DOTTED_VERSION]),
		'hint' => $l->t('To fix this issue update your libxml2 version and restart your web server.')
	);
}
```

There is no check which version is loaded!
"LIBXML_DOTTED_VERSION" is not equal "LIBXML_LOADED_VERSION"

if i edit "/lib/private/util.php" 
and change "LIBXML_DOTTED_VERSION" to "LIBXML_LOADED_VERSION" nextcloud started.

I have a Qnap NAS. There is only limited possible to install other software. 
so it will be also on devices from other manufacturers.

sorry, my english is to bad. i hope you understand me

**Comment by MorrisJobke** [comment]
I just checked this. I can't find any documentation for this constant in http://php.net/manual/en/libxml.constants.php

But a local test showed, that it is available:

```
    [LIBXML_DOTTED_VERSION] => 2.9.4
    [LIBXML_LOADED_VERSION] => 20904
```

It looks like the compiled version is older and a newer one is loaded. We also then need to adjust the if statement to something like:

```php
if(LIBXML_LOADED_VERSION >= 20700) {
```


@LukasReschke Does this makes sense? Should there be an additional check?

**Comment by kasi45** [comment]
"LIBXML_LOADED_VERSION"  is a new global constant since PHP 5.3
http://php.net/manual/en/migration53.global-constants.php

**Comment by MorrisJobke** [comment]
Thanks :) I will then try to fix this :)

**Comment by MorrisJobke** [comment]
Fix is in #612

**Comment by MorrisJobke** [comment]
fixed in master, stable10 and stable9
